Makefile

```diff
@@ -202,10 +173,35 @@ version: VERSION mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mo
 		rm $${i}.tmp; \
 	done
 
-install: dependencies core install-core
+install: core install-core
 
 install-linux-shortcuts: install-linux-scripts install-linux-icons install-linux-desktop
 
+install-win-dependencies:
+	@-echo "Installing OpenRA dependencies to $(DATA_INSTALL_DIR)"
+	@$(INSTALL_PROGRAM) soft_oal.dll "$(DATA_INSTALL_DIR)"
+	@$(INSTALL_PROGRAM) SDL2.dll "$(DATA_INSTALL_DIR)"
+	@$(INSTALL_PROGRAM) freetype6.dll "$(DATA_INSTALL_DIR)"
+	@$(INSTALL_PROGRAM) lua51.dll "$(DATA_INSTALL_DIR)"
+
+install-linux-dependencies:
+	ifneq ($(TARGETPLATFORM), "unix-generic")
+	@-echo "Installing OpenRA dependencies to $(DATA_INSTALL_DIR)"
+	@$(INSTALL_PROGRAM) soft_oal.so "$(DATA_INSTALL_DIR)"
+	@$(INSTALL_PROGRAM) SDL2.so "$(DATA_INSTALL_DIR)"
+	@$(INSTALL_PROGRAM) freetype6.so "$(DATA_INSTALL_DIR)"
+	@$(INSTALL_PROGRAM) lua51.so "$(DATA_INSTALL_DIR)"
+	else
+	@./thirdparty/configure-native-deps.sh
```

Comment:
We need to run this during compilation, not install! We can remove this `else` branch (but keep the `ifneq` above), and see my comment below.